backend/src/main/java/com/google/sps/notifs/EmailNotifier.java

```diff
@@ -58,21 +52,30 @@
    */
   private static final List<String> SCOPES = ImmutableList.of(GmailScopes.MAIL_GOOGLE_COM);
 
-  private static final String CREDENTIALS_FILE_PATH = "credentials.json";
+  private static final String CREDENTIALS_FILE_PATH =
+      "/home/grantjustice/IdeaProjects/Ad-lib/backend/credentials.json";
```

Comment:
This will need to be a path that's accessible when the project is deployed.

Reply:
+1 If we need a credential file to send email, maybe it should be inside this repo?

Reply:
Changed path